esp/services/ws_dfu/ws_dfuService.cpp

```diff
@@ -3488,10 +3504,17 @@ bool CWsDfuEx::doLogicalFileSearch(IEspContext &context, IUserDescriptor* udesc,
     ForEach(*it)
         addToLogicalFileList(it->query(), NULL, version, logicalFiles);
 
-    if (version >= 1.24)
-        resp.setCacheHint(cacheHint);
+    if ((oldCacheHint != cacheHint) && (totalFiles > maxFiles))
```

Comment:
I agree - I didn't know the warning was a popup (assumed it was displayed prominently at the top of the page or similar). 
Not sure popup is best mechanism - and think it _should_ be visible whichever page of the truncated results you're viewing.